examples/child-design-system/src/styles/base/_override.body.scss

```diff
@@ -0,0 +1,16 @@
+%ds-base {
+  font-family: $font-sans;
+  font-size: $base-font-size;
+  line-height: $base-line-height;
+}
+
+.ds-base {
+  @extend %ds-base;
+  color: $color-primary-alt;
```

Comment:
I couldn't get any of the scss files in `base` dir to do anything

Reply:
yeah, that's what i was seeing too with my testing. Moving the imports to core around changes things up, but then it causes other overrides to fail. I think in general our override process is super fragile we need to spend some time debugging it. I removed the overrides that didn't seem to work

Reply:
I agree 💯  we need to look into it deeper. I'm going to add something to the backlog for that